sps30-i2c/sps30.h

```diff
@@ -43,7 +43,8 @@ extern "C" {
 #define SPS30_I2C_ADDRESS 0x69
 #define SPS30_MAX_SERIAL_LEN 32
 #define SPS30_MEASUREMENT_DURATION_USEC 1000000 /* 1s measurement intervals */
-#define SPS30_RESET_DELAY_USEC 50000 /* 50ms delay after resetting the sensor */
+#define SPS30_RESET_DELAY_USEC 50000 /* 50ms delay after resetting the sensor  \
+                                      */
```

Comment:
WTF? I guess clang-format did this? It doesn't save a single byte on the line :confused:

Reply:
I thought so too.. but I'm not going to fight it